.github/workflows/test.yml

```diff
@@ -8,6 +8,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: hallee/eslint-action@master
+      - uses: hallee/eslint-action@v1.0.2
```

Comment:
@hallee probably want to leave this set to `master` for your own testing purposes, I think?